premake5.lua

```
@@ -0,0 +1,139 @@
+        pic "On"
+
+    -- Configuration-specific settings
+    filter "configurations:Debug"
+        defines { "DEBUG" }
+        symbols "On"
+
+    filter "configurations:Release"
+        defines { "NDEBUG" }
+        optimize "On"
+
+project "example"
```

Luacheck (1.2.0)

[warning] accessing undefined variable 'project'

(W113)